make.sh

```diff
@@ -17,7 +17,6 @@ rule() {
     clean)
       git clean -X -f
       dune clean
-    ;; gen) gen
```

Comment:
What was this even supposed to do?

Reply:
There used to be a `gen` bash function which it called, but that was removed way back in #455.